src/templates/base/base_chapter.html

```diff
@@ -65,14 +65,26 @@ <h2 id="explore-results">
       {{ self.explore_the_results() }}
     </a>
   </h2>
-  <a class="alt btn" href="https://discuss.httparchive.org/t/{{ metadata.get('discuss') }}">
-    <svg width="18" height="18" role="img">
+  <a class="alt btn cta" href="https://discuss.httparchive.org/t/{{ metadata.get('discuss') }}">
+    <svg width="22" height="22" role="img">
       <title>{{ self.discuss_this_chapter() }}</title>
       <use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#comment"></use>
     </svg>
     <span id="num_comments">{{ self.comments_fallback() }}</span> <span data-translation id="comment-singular">{{ self.comment() }}</span>
     <span data-translation id="comment-plural">{{ self.comments() }}</span>
   </a>
+  <a class="alt btn cta" href="https://github.com/HTTPArchive/almanac.httparchive.org/issues/new?assignees=&labels=bug%2C+writing&template=2020-content-issue.md&title=Issue+with+the+2020+%5BCHAPTER%5D+chapter">
+    <svg width="22" height="22" role="img">
+      <use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#github"></use>
+    </svg>
+    <span>Suggest Edit</span>
+  </a>
```

Comment:
An svg with `role="image"` must have an accessible name for screen readers. This is what is causing the Lighthouse failures as when you hide the text on mobile and only show the icons, there is no accessible name.

You should add a `<title>` element (which handily will also show as a hover tooltip).

Let's also put the full text in the `<title>` and make the actual text `aria-hidden`, so we get the full text even on mobile view (when text is hidden) but avoids repeating the text on desktop view (which text is not hidden):

```suggestion
  <a class="alt btn chapter-cta suggest-edit-cta" href="https://github.com/HTTPArchive/almanac.httparchive.org/issues/new?assignees=&labels=bug%2C+writing&template=2020-content-issue.md&title=Issue+with+the+2020+%5BCHAPTER%5D+chapter">
    <svg width="22" height="22" role="img">
      <title>GitHub: Suggest Edit</title>
      <use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#github"></use>
    </svg>
    <span aria-hidden="true">Suggest Edit</span>
  </a>
```

We should also move this text to each languages templates. But let's look at that after we've addressed all these issues.